File: 프로그래머스/unrated/181927. 마지막 두 원소/마지막 두 원소.js
function solution(num_list) {
    var length=num_list.length-1;
    if(num_list[length]>num_list[length-1]){
        num_list.push(num_list[length]-num_list[length-1]);
    }else{
        num_list.push(num_list[length]*2);
    }
    return num_list;
}

File: 프로그래머스/0/181927. 마지막 두 원소/마지막 두 원소.py
def solution(num_list):
    answer = []
    last=num_list[len(num_list)-1]
    prelast=num_list[len(num_list)-2]
    if last>prelast:
         num_list.append(last-prelast)
    else:
         num_list.append(last*2)
    return num_list
    